### ISSUE TITLE
Error with missing template after installing the Editorial theme on Grav CMS

### ISSUE THREAD

**Issue by nmorajda** [pmoreno-rodriguez/grav-theme-editorial#58]
Steps to Reproduce:

Installed Grav CMS using Composer.
Installed the "Editorial" theme using the command:

```
bin/gpm install editorial
```

The following dependencies were also prompted for installation:

    simplesearch
    shortcode-core

Edited `/user/config/system.yaml` to set the new theme:

    pages:
      theme: editorial

Observed Behavior: Upon loading the site, I encountered the following error:

`An exception has been thrown during the rendering of a template ("Template "forms/form.html.twig" is not defined.")`

Resolution: After installing the form plugin with:

```
bin/gpm install form
```

the error was resolved, and the theme functioned correctly.

Suggestion: It may be helpful to include the form plugin as a dependency or prompt its installation when setting up the Editorial theme.

**Comment by pmoreno-rodriguez** [comment]
I assume you installed the Grav Core package, without the Admin plugin. Is this correct?

I'll take a look at the dependencies in all scenarios, but that will be in a few weeks, I'm pretty busy right now.

Thanks for your suggestion.

**Comment by nmorajda** [comment]
Yes, that's correct. This installation was done without the Admin plugin.

**Comment by pmoreno-rodriguez** [comment]
Ok, @nmorajda. The latest version has plugin form as a dependency. I'm going to close this issue.
